cfme/fixtures/nuage.py

```diff
@@ -43,6 +44,27 @@ def create_basic_sandbox(nuage):
     box['l2_group'] = box['l2_domain'].create_child(
         nuage.vspk.NUPolicyGroup(name=random_name()))[0]
 
+    return create_namespace_from_enterprise(box)
+
+
+def create_namespace_from_enterprise(enterprise):
+    box = Namespace(**{key: {'obj': value} for key, value in enterprise.items()})
+    box.domain['num_subnets'] = '1'
```

Comment:
Does it have to be string? Perhaps we could update ValidateStats mixin to convert value to string?

Reply:
All the values in the mixin are now converted to strings.